site/en/docs/extensions/mv3/intro/index.md

```diff
@@ -1,51 +1,60 @@
 ---
 layout: 'layouts/doc-post.njk'
-
-# The page title. This appears at the top of the doc and as the page name
-# in Google Search.
 title: Welcome to Manifest V3
-
-# This appears below the title and is an optional teaser
 subhead: 'A step in the direction of security, privacy, and performance.'
-
-# This appears in the ToC of the project landing page at
-# /docs/[project-name]/. It also appears in the <meta description> used in 
-# Google Search.
 description: 'An introduction to Manifest V3'
-
-# The publish date
 date: 2020-11-09
+updated: 2022-09-02
```

Comment:
Did this edit change any technical content? The updated tag is only supposed to be used for that. This helps us surface changes developers should pay attention to vs those that merely correct text.

Reply:
The most important information added is the MV2 sunset timeline, which wasn't available when this article was first published. I consider this a significant addition. What do you think? 🤔